api/api_models/email_template.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel
+from enum import Enum
+
+from utils.enums import EmailTemplateName
+
+
+class EmailTemplateCreate(BaseModel):
+    template_name: EmailTemplateName
+    subject: str
+    html_content: str
+
+
+class EmailTemplateResponse(BaseModel):
+    id: int
+    template_name: str
+    subject: str
+    html_content: str
+
+    class Config:
+        orm_mode = True
```

Comment:
Pydantic has been upgraded so the orm_mode has been deprecated. The new value is from_attributes. You can check the Pydantic docs. Reference the version in the requirements.txt from the main branch